### PR TITLE
Testing 1.23.1~ynh2

### PR DESCRIPTION
**Pull request by OniriCorpe**
https://github.com/YunoHost-Apps/freshrss_ynh/pull/179
https://github.com/YunoHost-Apps/freshrss_ynh/pull/178

### CODE REVIEW

**Comment by OniriCorpe** [comment]
!testme

**Comment by OniriCorpe** [comment]
upgraded successfully on my ynh
no data loss 👌🏻

File: scripts/backup [hunk 1]
```diff
@@ -33,6 +33,12 @@ ynh_backup --src_path="/etc/nginx/conf.d/$domain.d/$app.conf"
 
 ynh_backup --src_path="/etc/php/$phpversion/fpm/pool.d/$app.conf"
 
+#=================================================
+# BACKUP LOGS
+#=================================================
+
+ynh_backup --src_path="/var/log/nginx/$domain-access.log"
```
Comment (alexAubin) [inline scripts/backup:40]: @OniriCorpe : uuuuh wut x_x I'm not sure to understand why we would backup/restore the nginx log of the entire domain x_X

**Comment by ericgaspar** [comment]
!testme

**Review by ericgaspar — APPROVED** [review]

**Pull request merged**